TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/Utils.java

```diff
@@ -70,119 +66,11 @@ public enum PacketPrintFormat {
         );
     }
 
-    public static <T> T setPrintStyleForCallable(PacketPrintFormat packetPrintFormat, Callable<T> r) throws Exception {
-        var oldStyle = printStyle.get();
-        printStyle.set(packetPrintFormat);
-        try {
-            return r.call();
-        } finally {
-            printStyle.set(oldStyle);
-        }
-    }
-
-    public static <T> T setPrintStyleFor(PacketPrintFormat packetPrintFormat, Supplier<T> supplier) {
-        try {
-            return setPrintStyleForCallable(packetPrintFormat, (()->supplier.get()));
-        } catch (RuntimeException e) {
-            throw e;
-        } catch (Exception e) {
-            throw new RuntimeException(e);
-        }
-    }
-
-    public static void setPrintStyleFor(PacketPrintFormat packetPrintFormat, Runnable r) {
-        setPrintStyleFor(packetPrintFormat, ()-> null);
-    }
-
-    public enum HttpMessageType { Request, Response }
-
-    public static String httpPacketBytesToString(HttpMessageType msgType, List<byte[]> packetStream) {
-        return httpPacketBytesToString(msgType, Optional.ofNullable(packetStream).map(p->p.stream())
-                .orElse(null));
-    }
-
-    public static String httpPacketBytesToString(HttpMessageType msgType, Stream<byte[]> packetStream) {
-        return httpPacketBufsToString(msgType, packetStream.map(bArr->Unpooled.wrappedBuffer(bArr)));
-    }
-
-    public static String httpPacketBufsToString(HttpMessageType msgType, Stream<ByteBuf> packetStream) {
-        packetStream = packetStream.map(bb->bb.duplicate()); // assume that callers don't want read-indices to be spent
```

Comment:
For my own curiosity and learning, I see that this file has been refactored to somewhere else, but this duplicate statement didn't, can you briefly explain why please?
Why don't we want that assumption anymore.

Reply:
I think I made this change because there were already some cases where callers didn't care (like when byte[]s were passed in).  I generally don't like assumptions, so it's now up to the caller to call duplicate if necessary.